daemon/algod/api/algod.oas2.json

```diff
@@ -2509,6 +2509,10 @@
           "protocol-version": {
             "description": "Protocol version is the protocol version Dryrun was operated under.",
             "type": "string"
+          },
+          "cost": {
+            "description": "Cost is the execution call of a stateful evaluation of an app call.",
```

Comment:
can this description be better? this description makes it sound like cost is an execution call.

Reply:
updated and move to `DryrunTxnResult`